packages/lesswrong/components/ea-forum/giving-portal/GivingSeasonBanner.tsx

```diff
@@ -291,13 +307,43 @@ const BannerDate: FC<{
   </div>
 );
 
+/**
+ * Convert from e.g. 1d to "1 day"
+ */
+function convertToLongFormat(time: string): string {
```

Comment:
I couldn't find a good place to put this

Reply:
`packages/lesswrong/lib/utils/timeUtil.ts`

Reply:
Oh nice, I must have only looked for a `dateUtil`